templates/plugin/python/module.conf.erb

```diff
@@ -7,14 +7,16 @@
   <%- configuration.sort.each do |key,value| -%>
   <%- if value.is_a?(Array) -%>
   <%- value.each do |v| -%>
-    <%= key %> <% if !!v == v %><%= v %><% else %>"<%= Shellwords.split(v).join('" "') %>"<% end %>
+    <%= key %> <% if !!v == v %><%= v %><% elsif v.is_a?(Numeric) %><%= v.to_s %><% else %>"<%= Shellwords.split(v).join('" "') %>"<% end %>
```

Comment:
it probably makes sense to convert this to epp

Reply:
See #819